### PR TITLE
Add color palette to tw config

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: web/tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,17 @@ module.exports = {
...
-        extend: {},
+        extend: {
+            colors: {
+                'brand-darkest-blue': '#12163B',
+                'brand-dark-blue': '#282768',
+                'brand-red': '#E81E25',
+                'brand-light-blue': '#6BABE5',
+                'brand-lightest-blue': '#ACC9E8',
+                'brand-purple': '#3F2B56',
+                'brand-light-purple': '#D4C0CD',
```
Comment (asasmith) [inline web/tailwind.config.js:13]: Let's keep to the number system `brand-1`. In the future if they change colors we won't have to worry about changing the variable name too.

**Review by asasmith — CHANGES_REQUESTED** [review]
Can you remove the package-lock changes? If you haven't added any packages these shouldn't be updating.

**Review by kcbreuil — COMMENTED** [review]
> Can you remove the package-lock changes? If you haven't added any packages these shouldn't be updating.

@asasmith there are no changes, i ran npm install so i could run the dev environment.

File: web/tailwind.config.js (hunk 1 above)
Comment (asasmith) [reply web/tailwind.config.js:13]: But then, in the future, when `brand-darkest-blue` gets changed to orange, we'll either have to live with the variable name `-blue` being associated with an orange color or do a "find and replace" on the whole project to update the variable name to something important. Neither option sounds good to me.

Comment (kcbreuil) [reply web/tailwind.config.js:13]: @asasmith Find and replace would still be more efficient than having to check every time I need to use a color 😅

**Pull request merged**